public/assets/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 //
 // Main JavaScript file
 // --------------------------------------------------------
+var PLTRFM = {
```

Comment:
Looks like you have this misspelled in a lot of places (I also misspelled it in some places). Let's make sure we are using the correct name everywhere: "PLTFRM".